MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include CHANGELOG.md
+include LICENSE
+include requirements*.txt
```

Comment:
We use poetry, so there are no requirements.txt files.

Reply:
Removed.
The resulting tar.gz content, looks right to me. Wasnt able to install it as we have too old dependencies, will work on that next.
![image](https://user-images.githubusercontent.com/30687616/233369527-d5df8ac8-ad95-4610-bda5-52d85dc924df.png)

Reply:
Installed it into an empty venv and seemed to worked as i was able to import it